src/Command/Setup.php

```diff
@@ -57,7 +57,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io->setDecorated(true);
         $io->comment(sprintf('Composing your <info>%s</info> environment', (string) $envType));
 
-        $vars = new Vars($io->ask('Vendor name', null, [$this, 'validateVar']), $io->ask('App name', null, [$this, 'validateVar']));
+        $vars = new Vars(
+            $io->ask('Vendor name', null, [$this, 'validateVar']),
```

Comment:
Calling a constructor multi lined is often better for readability, when the number of arguments exceeds a certain amount, or like in this case, where the arguments are not pre-populated in tmp vars.